internal/net/grpc/status/status_test.go

```diff
@@ -21,10 +21,11 @@ import (
 	"reflect"
 	"testing"
 
-	grpcerr "github.com/vdaas/vald/apis/grpc/errors"
-	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/apis/grpc/errors"
```

Comment:
Does this package cover both `grpc/error` and `internal/error`?

Reply:
for the test, it's covered but for the main implementation, it's not enough.

Reply:
Well, but it seems this test will fail with `build failed` because of `undefined: errors.Errorf` and `undefined: errors.Is`

Reply:
oh, misunderstand,  "github.com/vdaas/vald/internal/errors" only is better

Reply:
I see.
But we also use `grpc/errors` too, for example [this line](https://prod.liveshare.vsengsaas.visualstudio.com/join?912B2EF8475A080C89ED75FAC63DB0765339 )